.eslintrc.js

```diff
@@ -5,6 +5,8 @@
  *   https://github.com/reactorcore/eslint-config-hackreactor
  */
 
-module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
-};
\ No newline at end of file
+// module.exports = {
+//   extends: './node_modules/eslint-config-hackreactor/index.js'
+// };
+
+module.exports = { “extends”: “airbnb-base” };
```

Comment:
what's airbnb-base? should i be using this instead of plain "extends": "airbnb"?